packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js

```diff
@@ -322,4 +322,67 @@ export default class DedicatedServerDashboard {
       type: 'navigation',
     });
   }
+
+  onChangeIntervention(noIntervention) {
+    const { monitored } = this.server;
+    this.updatingNoIntervention = true;
+    this.Server.updateMonitoring(
+      this.$stateParams.productId,
+      monitored,
+      noIntervention,
+    )
+      .then(() => {
+        this.server.noIntervention = noIntervention;
+        if (noIntervention === true) {
+          this.Alerter.success(
+            this.$translate.instant(
+              'server_configuration_no_intervention_activate_success',
+              {
+                serverName: this.server.displayName,
+              },
+            ),
+            'server_dashboard_alert',
+          );
+        } else {
+          this.Alerter.success(
+            this.$translate.instant(
+              'server_configuration_no_intervention_deactivate_success',
+              {
+                serverName: this.server.displayName,
+              },
+            ),
+            'server_dashboard_alert',
+          );
+        }
+      })
+      .catch((err) => {
+        this.server.noIntervention = !noIntervention;
```

Comment:
Related to comment on L335, `this.server.noIntervention` is `undefined`, and is set to opposite param if call fails ?

Reply:
oui-switch component sets "this.server.noIntervention" value to true or false. In case API fails to update the same value, it needs to be reverted to the previous value.